src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -51,26 +57,48 @@ const AssetList: FunctionComponent<AssetListProps> = ({
   className,
   featureTypes,
 }) => {
-  const { id, type, isReported } = selection
+  const { id, type, isReported, isChecked } = selection
   const { description, icon }: Partial<FeatureType> =
     featureTypes?.find(({ typeValue }) => typeValue === type) ?? {}
 
   const label = [description, id].filter(Boolean).join(' - ')
 
   if (!id) return null
 
+  const reportedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'reported'
+  ) as ReportedFeatureType
+
+  const checkedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'checked'
+  ) as CheckedFeatureType
+
```

Comment:
Both expressions can be `undefined`, but the code doesn't reflect that

Reply:
✅